app/views/users/registrations/new.html.erb

```diff
@@ -2,28 +2,27 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Welcome!</h1>
-    <p class="govuk-body">
-      We are glad you would like to sign up your organisation for GovWifi.
-    </p>
-    <p class="govuk-body">
-      You have the option to sign up for a three month trial. After you have
-      tested GovWifi, you can go live and scale it at your organisation for free!
+    <h1 class="govuk-heading-l">Sign up your Organisation to GovWifi</h1>
+
+    <p class="govuk-!-padding-bottom-1">
+      <%= link_to "If you are trying to connect to GovWifi as an end user, see instructions here", "", class: "govuk-link govuk-!-font-size-19" %>
     </p>
 
     <div class="govuk-grid-column-two-thirds govuk-!-padding-left-0">
       <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
 
         <div class="govuk-form-group <%= "govuk-form-group--error" if resource&.errors&.any? %>">
-          <%= f.label :email, "Provide your email address", class: "govuk-label"  %><br />
+          <%= f.label :email, "Enter your email address", class: "govuk-label"  %><br />
+          <div class="govuk-hint"> Must be a valid government email address </div>
           <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "govuk-input", required: true %>
         </div>
-
         <div class="actions">
           <%= f.submit "Sign up", class: "govuk-button govuk-!-margin-top-4" %>
         </div>
       <% end %>
 
+      <%= link_to "Forgot password", new_password_path(resource_name), class: "govuk-link govuk-!-font-size-19" %><br />
```

Comment:
This already exists and can be displayed dynamically from the `users/shared/links` partial.

Reply:
Yeah, I looked in the partial, saw `controller_name !=` and ran away. 

I'm worried about how much time I have for sorting out these pages today, but I can try and fix this if you think it's best?

Reply:
Talked about this IRL and Tom pointed out the very easy solution - nice one Tom!